components/filemanager/overview.md

```diff
@@ -195,133 +194,133 @@ The <a href = "https://www.telerik.com/blazor-ui/file-manager" target="_blank">B
-                DateModifiedUtc = new DateTime(2022, 2, 13),
-                Path = Path.Combine(Images.Path, "Dashboard Design.png"),
-                Size = 1024
-            };
+        {
+            Id = "6",
+            ParentId = Images.Id,
+            Name = "Dashboard Design",
+            IsDirectory = false,
+            HasDirectories = false,
+            Extension = ".png",
+            DateCreated = new DateTime(2022, 1, 10),
+            DateCreatedUtc = new DateTime(2022, 1, 10),
+            DateModified = new DateTime(2022, 2, 13),
+            DateModifiedUtc = new DateTime(2022, 2, 13),
+            Path = Path.Combine(Images.Path, "Dashboard Design.png"),
+            Size = 1024
+        };
 
         var gridDesign = new FlatFileEntry()
-            {
-                Id = "7",
-                ParentId = Images.Id,
-                Name = "Grid Design",
-                IsDirectory = false,
-                HasDirectories = false,
-                Extension = ".jpg",
-                DateCreated = new DateTime(2022, 1, 12),
-                DateCreatedUtc = new DateTime(2022, 1, 12),
-                DateModified = new DateTime(2022, 2, 13),
-                DateModifiedUtc = new DateTime(2022, 2, 13),
-                Path = Path.Combine(Images.Path, "Grid Design.jpg"),
-                Size = 1024
-            };
+        {
+            Id = "7",
+            ParentId = Images.Id,
+            Name = "Grid Design",
+            IsDirectory = false,
+            HasDirectories = false,
+            Extension = ".jpg",
+            DateCreated = new DateTime(2022, 1, 12),
+            DateCreatedUtc = new DateTime(2022, 1, 12),
+            DateModified = new DateTime(2022, 2, 13),
+            DateModifiedUtc = new DateTime(2022, 2, 13),
+            Path = Path.Combine(Images.Path, "Grid Design.jpg"),
+            Size = 1024
+        };
 
         var files = new List<FlatFileEntry>()
-            {
-                workFiles,
+        {
+            workFiles,
 
-                Documents,
-                specification,
-                report,
+            Documents,
+            specification,
+            report,
 
-                Images,
-                dashboardDesign,
-                gridDesign
-            };
+            Images,
+            dashboardDesign,
+            gridDesign
+        };
 
         return await Task.FromResult(files);
     }
 }
 ````
 
->The `Path` of each `FileManagerEntry` should end with the `Name` of the data item, mirroring the format used in actual file systems.
+>Each `FileManagerEntry`'s `Path` needs to finish with the data item's `Name`, following the style used in real file systems. Also, there should be a constant root path as the topmost path segment to show the Breadcrumb right. If there's no root path given, we consider the first folder is the root. However, this causes an issue because the Breadcrumb doesn't recognize it and shows the `Home` icon instead.
```

Comment:
Perhaps, we should avoid adding specific details about the discussed problem, as there might be other issues beyond just the breadcrumb. 

Additionally, the root path segments is important not just because the breadcrumb. Clients can have more than one 'root' directory and can change it by adjusting the 'Path' parameter value of the FileManager. 

For instance, you can bind the FileManager to a collection of files with different topmost 'root' directories (for example 'C:/...' and 'D:/...'). Thus you can programmatically change the `Path` of the FileManager from 'C:/...' to 'D:/...' to display other files.

===

In my opinion, adding a brief sentence emphasizing that the topmost file's path should include a root path at the start is enough.

Reply:
Done